### ISSUE TITLE
Docs problem with introduction

### ISSUE THREAD

**Issue by bradh** [DHI-GRAS/terracotta#146]
https://terracotta-python.readthedocs.io/en/latest/index.html says:

```
It is trivial to get going. Got a folder full of cloud-optimized GeoTiffs in different projections you want to have a look at in your browser? terracotta serve -p {name}.tif and terracotta connect localhost:5000 get you there.
```

I found it not-so-trivial. 

```
$ terracotta serve -p 2018-10-02.tif 
Error: no such option: -p
```

More generally, I couldn't find a way to serve up just one (or any number) of true-colour sentinel images I had.

```
bradh@kubuntu1804:~/terracotta-test/Sentinel2$ ls -al *.tif
-rw-rw-r-- 1 bradh bradh  39956894 Jun 23 20:50 2018-10-02.tif
-rw-rw-r-- 1 bradh bradh  46664331 Jun 23 20:50 2018-10-05.tif
-rw-rw-r-- 1 bradh bradh 117639558 Jun 23 20:50 2018-10-07.tif
-rw-rw-r-- 1 bradh bradh 129014822 Jun 23 20:50 2018-10-10.tif
-rw-rw-r-- 1 bradh bradh  18720783 Jun 23 20:50 2018-10-12.tif
-rw-rw-r-- 1 bradh bradh 129643047 Jun 23 20:50 2018-10-15.tif
-rw-rw-r-- 1 bradh bradh  74030115 Jun 23 20:50 2018-10-17.tif
-rw-rw-r-- 1 bradh bradh 139422833 Jun 23 20:50 2018-10-20.tif
-rw-rw-r-- 1 bradh bradh  70036030 Jun 23 20:50 2018-10-25.tif
-rw-rw-r-- 1 bradh bradh 135436629 Jun 23 20:50 2018-10-27.tif
-rw-rw-r-- 1 bradh bradh 134728093 Jun 23 20:50 2018-10-30.tif
-rw-rw-r-- 1 bradh bradh 136648909 Jun 23 20:50 2018-11-01.tif
-rw-rw-r-- 1 bradh bradh  79179173 Jun 23 20:50 2018-11-04.tif
-rw-rw-r-- 1 bradh bradh  53509706 Jun 23 20:50 2018-11-06.tif
-rw-rw-r-- 1 bradh bradh  33364421 Jun 23 20:50 2018-11-09.tif
-rw-rw-r-- 1 bradh bradh 130338356 Jun 23 20:50 2018-11-11.tif
-rw-rw-r-- 1 bradh bradh 120515754 Jun 23 20:50 2018-11-14.tif
bradh@kubuntu1804:~/terracotta-test/Sentinel2$ terracotta ingest {date}.tif -o real.sqlite
Usage: terracotta ingest [OPTIONS] RASTER_PATTERN

Error: Invalid value for "raster-pattern": Given pattern matches no files
bradh@kubuntu1804:~/terracotta-test/Sentinel2$ terracotta ingest /home/bradh/terracotta-test/Sentinel2/{date}.tif -o real.sqlite
Usage: terracotta ingest [OPTIONS] RASTER_PATTERN

Error: Invalid value for "raster-pattern": Given pattern matches no files
```

**Comment by dionhaefner** [comment]
> I found it not-so-trivial.

While I am thankful that you reported this usability issue, please cut out the snark. Remember that this, as most open-source projects, is driven by people volunteering to put in more work than they are obligated to.

That being said, the issue here is two-fold:

- A bug in the documentation that I will fix ASAP - the flag is called `-r`, not `-p`.
- A misconception on your side how the raster patterns work. They only consume alphanumeric characters, not special characters such as `_` or (in your case) `-`. So to make this work, either rename your files to something like `20181114.tif`, change the pattern to `{year}-{month}-{day}.tif`, or use the Python API for ingestion.

If you have any suggestion on how to formulate this clearer in the documentation I would be happy to hear it.

**Comment by bradh** [comment]
My sincere apologies on the way I expressed it - it was meant to be light-hearted, not snarky. I am truly sorry that is how it came across. I have experienced the effect that "entitled" users have on the morale of an open source project, and would mich to be part of the "thanks for all your work" team rather than the "why doesn't your project work for my niche case" detractors. So thanks for the quick response over the weekend!

I can't unsay it, but I can (and should) apologize again: I'm sorry for the poor choice of words and resulting miscommunication.

I'll close this ticket and provide a follow-up PR with some documentation suggestions.
